### PR TITLE
Add playstore.coffee to hubot-scripts

### PR DESCRIPTION
**Pull request by koomar**
@siminm @Nextdoor/engineering-opensource-collaborators

### CODE REVIEW

**Comment by siminm** [comment]
:+1:

File: playstore.coffee [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Description:
+#   All things Google's Playstore
+#
+#   Set your Google Playstore application id as an
+#   environment variable `PLAYSTORE_APP_ID`
+#   e.g. PLAYSTORE_APP_ID=com.nextdoor
+#
+# Dependencies:
+#   cheerio
```
Comment (mikhail) [inline playstore.coffee:9]: So what about these dependencies? Should there be a requirements.json?

File: playstore.coffee [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+#
+# Commands:
+#   hubot playstore version - What is the latest version?
+#   hubot playstore rating - What is the rating version?
+#   hubot playstore specs - Show all specs about the current Android App.
+#
+# Authors:
+#   Abhijeet Kumar (abhijeet@nextdoor.com)
+#   Jeff Robidoux (jeff@nextdoor.com)
+#   Daisuke Fujiwara (daisuke@nextdoor.com
```
Comment (mikhail) [inline playstore.coffee]: Close parenthesis

File: playstore.coffee [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+cheerio = require('cheerio')
+
+getValue = (body, selector) ->
```
Comment (mikhail) [inline playstore.coffee]: `getText` would be more accurate

File: playstore.coffee [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                "text": "*Version:* #{version}\n"+
+                        "*Date Published:* #{datePublished}\n"+
+                        "*Rating:* #{score}/5\n"+
+                        "*Number of reviews:* #{numReviews}\n"+
+                        "*Installs:* #{numDownloads}\n"+
+                        "*Min Android Version:* #{minOS}\n"+
+                        "*File Size:* #{fileSize}"}
+        }
+
+  robot.on "playstore:new_version_notification", (data) ->
```
Comment (mikhail) [inline playstore.coffee:96]: When does this fire off?

File: playstore.coffee [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
+  robot.respond /(.*) playstore version/i, (msg) ->
+    msg.http(getPlaystoreURL())
+       .get() (err, res, body) ->
+         msg.send "Current live version of Android is v#{getValue body, 'div[itemprop=softwareVersion]'} in Google Playstore."
+
+  robot.respond /(.*) playstore rating/i, (msg) ->
+    msg.http(getPlaystoreURL())
+       .get() (err, res, body) ->
+         msg.send "Nextdoor Android has a score of #{getValue body, '.score'}/5 with #{getValue body, '.reviews-num'} total reviews."
```
Comment (mikhail) [inline playstore.coffee]: This is Nextdoor specific. Make it generic

File: playstore.coffee (hunk 1 above)
Comment (koomar) [inline playstore.coffee:9]: Good question - not sure how to best approach this. One part of me just wants to open source the whole repo. Perhaps, for now, let's publish one by one

File: playstore.coffee (hunk 2 above)
Comment (koomar) [inline playstore.coffee]: good catch

File: playstore.coffee (hunk 3 above)
Comment (koomar) [inline playstore.coffee]: Aye

File: playstore.coffee (hunk 4 above)
Comment (koomar) [inline playstore.coffee:96]: Cron job - i will add a document.

File: playstore.coffee (hunk 5 above)
Comment (koomar) [inline playstore.coffee]: Ouch. good catch

**Comment by mikhail** [comment]
:shipit:

**Pull request merged**